api/src/Controller/BaseController.php

```diff
@@ -75,7 +81,15 @@ public function processIncludes(Request $request, Response $response, $args, $in
 
     public function __invoke(Request $request, Response $response, $args)
     {
-        $result = $this->buildResource($request, $response, $args);
+        try {
```

Comment:
SonarCloud also complains about the complexity of this method now. I see a way to refactor it, and will tackle that later.